### PR TITLE
Release candidate v1.0.0

### PR DESCRIPTION
**Pull request by BarryPiccinni**
Adds all the necessary changes to CW to make the application compatible with template v1.0.0.
Changes made:

- Bumped version to 1.0.0
- Updated package.json to user q-transformer v1.0.0
- Refactored removeUnwantedHiddenAnswers function to remove unwanted answers from checkbox questions
- Added new IDs to the ID Mapper
- Updated UISchema to include:
  - New CYA structure
  - New answers in answer lookup
  - Added additional component to contact preference radio button
  - Added new questions
- Updated tests to reflect new formHelper functionality

All unit tests and integration tests are passing. App has been manually clicked through and working as expected.

### CODE REVIEW

File: questionnaire/questionnaireUISchema.js [hunk 1]
```diff
@@ -523,5 +560,54 @@ module.exports = {
...
+            }
+        }
+    },
+    'p-gp-enter-your-address': {
+        options: {
+            properties: {
+                'q-gp-building-and-street': {
+                    options: {
+                        macroOptions: {
+                            classes: ''
```
Comment (armoj) [inline questionnaire/questionnaireUISchema.js:592]: is this to get rid of the default widths applied to them?

File: questionnaire/questionnaireUISchema.js [hunk 2]
```diff
@@ -523,5 +560,54 @@ module.exports = {
...
+                        }
+                    }
+                },
+                'q-gp-building-and-street-2': {
+                    options: {
+                        macroOptions: {
+                            classes: ''
```
Comment (armoj) [inline questionnaire/questionnaireUISchema.js:599]: same question as above?

File: questionnaire/questionnaireUISchema.js (hunk 1 above)
Comment (BarryPiccinni) [reply questionnaire/questionnaireUISchema.js:592]: Yes. This is the same as the page for the applicants address, the width of the top to lines doesn't match the size derived from the character count, but this was requested for aesthetic reasons.

File: questionnaire/questionnaireUISchema.js (hunk 2 above)
Comment (BarryPiccinni) [reply questionnaire/questionnaireUISchema.js:599]: As above.

**Pull request merged**